src/components/tabs/index.tsx

```diff
@@ -0,0 +1,204 @@
+import {
+  KeyboardEvent,
+  KeyboardEventHandler,
+  ReactElement,
+  useLayoutEffect,
+  useRef,
+  useState,
+} from 'react'
+import { useId } from '@react-aria/utils'
+import slugify from 'slugify'
+import Tab, { TabProps } from './components/tab'
+import { TabsProps } from './types'
+import s from './tabs.module.css'
+import classNames from 'classnames'
+
+const Tabs = ({
+  ariaLabel,
+  ariaLabelledBy,
+  children,
+  initialActiveIndex = 0,
+  showAnchorLine = true,
+}: TabsProps): ReactElement => {
+  /**
+   * Disallow rendering a Tabs component with only one Tab.
+   */
+  if (!Array.isArray(children)) {
```

Comment:
Totally non-blocking enhancement, but if we want we could validate that children of `Tabs` are all `Tab` components. We do [something like this in CodeTabs](https://github.com/hashicorp/react-components/blob/963d39ac51d63ff45a8a85ab078c7130502c972d/packages/code-block/partials/code-tabs/index.js#L47), which might be useful as a reference.

Reply:
I think this is a great idea, I'll take a look at that example and try it out!

Reply:
Updated in: https://github.com/hashicorp/dev-portal/pull/104/commits/5477198bd086bfc3789aa44464bd98481e90eded